WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -942,6 +942,18 @@ public boolean isNewPost() {
         return mIsNewPost;
     }
 
+    /*
+     * returns true if the user has permission to publish the post - assumed to be true for
+     * dot.org sites because we can't retrieve their capabilities
```

Comment:
Just tested with a Contributor role on a .org site. The UX is fine (user gets a error after publishing, and error message looks good)
<img width="415" alt="screen shot 2017-06-20 at 18 41 12" src="https://user-images.githubusercontent.com/40213/27344857-1ef49288-55e8-11e7-99a3-3740e38f36e1.png">